subsys/logging/log_msg.c

```diff
@@ -63,10 +63,12 @@ void z_impl_z_log_msg_static_create(const void *source,
 					    NULL, 0, flags,
 					    strl, ARRAY_SIZE(strl));
 
-		if (len > Z_LOG_MSG_MAX_PACKAGE)
-		{
-			LOG_WRN("Log message dropped because it exceeds current limitation (%u)",
-				(uint32_t)Z_LOG_MSG_MAX_PACKAGE);
+		if (len > Z_LOG_MSG_MAX_PACKAGE) {
+			struct cbprintf_package_hdr_ext *pkg =
+				(struct cbprintf_package_hdr_ext *)package;
+
+			LOG_WRN("Message (\"%s\") dropped because it exceeds size limitation (%u)",
```

Comment:
Is it possible that the format string itself is too long? If so, what would happen here?

Reply:
Format string is in rodata so it is not included in the package.

Reply:
OK, thanks for clarifying

Reply:
@nordic-krch But how does being in rodata is changing anything? What am I missing? The `%s` formatter is being used here and thus if the `pkg->fmt` is larger than the limit, would it trigger the issue? Could this lead to endless recursive call?